composer.json

```diff
@@ -29,16 +39,15 @@
             "Tests\\": "tests"
         }
     },
-    "extra": {
-        "laravel": {
-            "providers": [
-                "BladeUI\\Heroicons\\BladeHeroiconsServiceProvider"
-            ]
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/mallardduck/blade-icons"
         }
-    },
-    "config": {
-        "sort-packages": true
-    },
+    ],
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+    "scripts": {
+        "generate": "./bin/generate"
+    }
```

Comment:
Can you undo the re-sorting of keys here?

Reply:
yeah, sorry I applied composer normalize after adding the scripts.